### ISSUE TITLE
Restart OSX before Security Settings are registered?

### ISSUE THREAD

**Issue by jasonm23** [kasper/phoenix#55]
I've seen this happen a few times now, hopefully it's reproducible.

(OS Yosemite 10.10.5)

1. Remove Phoenix from Security/Accessibility
2. Build new App Bundle
3. Run, follow security notice to System Prefs (and Quit phoenix dialog)
4. Add Phoenix in Security/Accessiblity
5. Start Phoenix
6. process starts again at (3) and Phoenix will not run until OSX is restarted. (have not yet tried just a logout)

**Comment by jasonm23** [comment]
Steps to reproduce do not work, I think I can close this.  @kasper any sign of this one for you.

I am putting it down to something left over from a previous build, but it's pretty odd.

**Comment by kasper** [comment]
This seems to happen occasionally if you have recently ran a build from Xcode. Once in a while, this seems to require a reauthorisation for the Accessibility API. Not sure why though. Perhaps this is an Xcode issue? The app should be fine in basic use though.

**Comment by jasonm23** [comment]
Ok closing
